client/src/api.ts

```diff
@@ -25,29 +24,18 @@ const DIALOG_ENDPOINT =
 const GRAPHQL_ENDPOINT = process.env.GRAPHQL_ENDPOINT || "/graphql/";
 
 export async function fetchLesson(lessonId: string): Promise<Lesson> {
-  const API_SECRET = await getApiKey();
```

Comment:
Remove getApiKey from config.ts (maybe just delete config.ts?)

Reply:
that's on the 'old' side of the diff.

Reply:
app.js is still sending API_SECRET to /config